spec/03-plugins/17-jwt/01-jwt_parser_spec.lua

```diff
@@ -29,6 +29,53 @@ describe("Plugin: jwt (parser)", function()
         ]], true), token)
       end
     end)
+    it("should properly encode using HS384", function()
+      local token = jwt_parser.encode({
+        name  = "John Doe",
+        admin = true,
+        sub   = "1234567890"
+      }, "secret", 'HS384')
```

Comment:
We consistently use double quotes throughout this codebase, will you please convert these single ones? There are other similar places in this patch. Thanks!